apm-protocol/apm-ui-protocol/src/main/resources/ui-graphql/README.md

```diff
@@ -0,0 +1,34 @@
+# Abstract
+**apm-ui-protocol** declares all services, using GraphQL API style, which provide by Collector UI module.
+
+## Services
+### [Common](common.graphqls)
+
+Include common objects, which used in global
+
+### [Overview Layer Service](overview-layer.graphqls)
+
+Query data without specific application, server or service. It includes info for overview the whole cluster.
+
+### [Application Layer Service](application-layer.graphqls)
+
+Query application related data with specific application code.
+
+### [Server Layer Service](server-layer.graphqls)
+
+Query server related data with specific server id.
+
+### [Service Layer Service](service-layer.graphqls)
+
+Query service related data with specific service id
+
+### [Trace Service](trace.graphqls)
+
+Query trace by some conditions.
+
+### [Alarm Service](alarm.graphqls)
+
+Query alarm info.
+
+## Version
+1.0-alpha
```

Comment:
With reference to [REST API Versioning](http://searchmicroservices.techtarget.com/definition/RESTful-API) , I prefer to use `v1alpha`

Reply:
@hanahmily Can you address the directly link? or provide a screenshot?

Reply:
For instance `http://collector.host/graphql/v1alpha/`

Reply:
@hanahmily The **REST API Versioning** document provides nothing about how to version.

![image](https://user-images.githubusercontent.com/5441976/34454420-15463f42-eda6-11e7-9bd5-9e25e2fb18ba.png)

Reply:
For the `*.graphqls` file name format ,I prefer to use [k8s style](https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec) @wu-sheng